com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/ApplicationConfiguration.java

```diff
@@ -123,6 +124,7 @@
     public static final Integer DEFAULT_FLOWABLE_JOB_EXECUTOR_MAX_THREADS = 32;
     public static final Integer DEFAULT_FLOWABLE_JOB_EXECUTOR_QUEUE_CAPACITY = 16;
     public static final Integer DEFAULT_FSS_CACHE_UPDATE_TIMEOUT_MINUTES = 30;
+    public static final Integer DEFAULT_THREAD_MONITOR_UPDATE_IN_MILIS = 100;

```

Comment:
Isn't this too frequent? Doing a full thread dump probably has a significant cost. At least I would assume so.